### ISSUE TITLE
params for render_to_string get into the wkhtmltopdf call

### ISSUE THREAD

**Issue by fwoeck** [igor-alexandrov/wisepdf#3]
Hi Igor, when I use a layout in my render methods, like:

```
          pdf = render_to_string pdf:      'Document-Notes',
                                 template: 'dokuments/dokuments_notes',
                                 layout:   'meeting_report'
```

it get's into the wkhtmltopdf-call and causes an error. Same for the template:

```
"/bin/wkhtmltopdf" "--encoding" "UTF-8" "--layout" "meeting_report" "--dpi" "100" "--page-size" "A4" "--margin-top" "0" "--margin-bottom" "0" "--margin-left" "0" "--margin-right" "0" "--template" "meetings/generate_report" "--quiet" "-" "-"

Unknown long argument --layout
```

Thanks in advance,
Frank

**Comment by igor-alexandrov** [comment]
Thanks!
Fixed in wisepdf-1.1.2.

**Comment by fwoeck** [comment]
Thank you!
